README.md

```diff
@@ -16,7 +16,12 @@ Solve them once and for all.
 INSTALL
 -------
 
-    $ gem install git-up
+    $ ~~gem install git-up~~
+
+    $ git clone git@github.com:milosm/git-up.git
+    $ cd git-up && gem build git-up.gemspec
+    $ sudo gem install git-up-0.5.12.1.gem
+    (Version may vary)
```

Comment:
Until it's not merged yes (needed clear instructions for people that wanted to install in before), after the pull request will be merged then no it will not be needed anymore.